charts/kube-prometheus-stack/Chart.yaml

```
@@ -22,7 +22,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 41.0.0
+version: 41.0.1
```

new feature should raise minor verison

```suggestion
version: 41.1.0
```